src/super_gradients/training/transforms/transforms.py

```diff
@@ -1079,6 +1079,108 @@ def get_equivalent_preprocessing(self) -> List:
         return []
 
 
+@register_transform(Transforms.DetectionRandomSideCrop)
+class DetectionRandomSideCrop(DetectionTransform):
+    """Preprocessing transform to crop in width an image and bboxes from the border.
+    This is, the output image will have a width between `min_rel_width` and `max_rel_width` of the original image.
+    
+    Note: It assumes the targets are in (X,Y,X,Y,label) format.
+    """    
+
+    def __init__(self, min_rel_width:float = 0.3, max_rel_width: float = 0.6,  p_side_right: float = 0.5, prob: float = 1.0):
+        """_summary_
+
+        :param min_rel_width: minimum relative width of the resulting crop, defaults to 0.3
+        :param max_rel_width: maximum relative width of the resulting crop, defaults to 0.6
+        :param p_side_right: probability of keeping the right side when croping, defaults to 0.5
+        :param prob: probability of applying the transformation, defaults to 1.0
+        :raises AssertionError: Input parameters are not in the correct range
+        """     
+
+        assert 0 < min_rel_width <= 1, f"`min_rel_width` value must be between 0 (not included) and 1, found {min_rel_width}"
```

Comment:
Please try to avoid using asset statements to check input arguments.
Instead use exceptions to raise a error is some argument is invalid.
Assert statements should stay in unit tests :)

https://www.inspiredpython.com/article/assertions-and-exceptions-are-not-the-same